src/expressions/functions/string.ts

```diff
@@ -23,6 +23,17 @@ import { normalizeExpr } from "../../utils/normalize-variable";
 
 import { CypherFunction } from "./CypherFunctions";
 
+/**
+ * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/functions/string/)
+ * @group Cypher Functions
+ * @category String
+ */
+export function btrim(original: string | Expr, trimCharacter?: string | Expr): CypherFunction {
+    const normalizedOriginal = normalizeExpr(original);
+    const normalizedTrimCharacter = trimCharacter ? normalizeExpr(trimCharacter) : undefined;
+    return new CypherFunction("btrim", filterTruthy([normalizedOriginal, normalizedTrimCharacter]));
+}
+
 /**
  * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/functions/string/)
```

Comment:
Sorry unrelated but URL is not precise also for the other functions
```suggestion
 * @see [Cypher Documentation](https://neo4j.com/docs/cypher-manual/current/functions/string/#functions-left/)
```